autonomi/src/networking/mod.rs

```diff
@@ -80,6 +83,7 @@ pub enum NetworkError {
 #[derive(Debug, Clone)]
 pub struct Network {
     task_sender: tokio::sync::mpsc::Sender<NetworkTask>,
+    _driver_task: Arc<JoinHandle<()>>,
```

Comment:
Why keep this one around?

Reply:
Removed it as I didn't observe any difference. Although made the task receiver an `Arc` so we never drop the original receiver (this fixed the dropped receiver error).

Reply:
Was a quick fix for the dropped network client error.

This was mostly an experimental branch to figure out why we didn't get any quote responses from non-local nodes